opal/mca/accelerator/cuda/accelerator_cuda_component.c

```diff
@@ -115,30 +119,34 @@ static int accelerator_cuda_component_register(void)
     return OPAL_SUCCESS;
 }
 
-static opal_accelerator_base_module_t* accelerator_cuda_init(void)
+int opal_accelerator_cuda_delayed_init()
 {
-    int retval, i, j;
-    CUresult result;
+    CUresult result = OPAL_SUCCESS;
```

Comment:
Wrong type (CUresult is an enum that we don't control and we  shouldn't mix enum and non-enum values)

```suggestion
    int result = OPAL_SUCCESS;
```

Reply:
Done